doc/manual/manual.tex

```diff
@@ -20,56 +20,49 @@
 
-This document describes how to install the Arcade Learning Environment as well as how to use it.
-We will present the different ways to implement an agent: 
+This document is roughly divided into three parts. The first part describes how to install the Arcade Learning Environment. The second part describes the various ALE interfaces currently available: 
 \begin{enumerate}
-  \item \textbf{Shared Library interface} (C++): Loads ALE as a shared library (Section 
-  \ref{sec:shared_library_interface}).}
-  \item \textbf{CTypes interface} (Python): Uses the ALE as an external Python library, 
-  allowing one to write ALE code by just importing the corresponding library. This is a very
-  fast interface (Section \ref{sec:python_interface}).
-  \item \textbf{FIFO interface} (any language): Communicates with ALE through a text interface 
-  using \verb+stdin/stdout+ (Section \ref{sec:pipes_interface}).}
-  \item \textbf{RL-Glue interface} (C/C++, Java, Python, Matlab or Lisp): Communicates
-  with ALE via RL-Glue (Section \ref{sec:rlglue_interface}).}
+  \item \textbf{Shared Library interface} (C++ only): Loads ALE as a shared library (Section 
+  \ref{sec:shared_library_interface}).
+  \item \textbf{CTypes interface} (Python only): A fast Python interface to ALE, provided as a Python package (Section \ref{sec:python_interface}).
+  \item \textbf{FIFO interface} (all languages): Communicates with ALE through a text interface (Section \ref{sec:pipes_interface}).
+  \item \textbf{RL-Glue interface} (C/C++, Java, Python, Matlab, Lisp, Go): Communicates with ALE via RL-Glue (Section \ref{sec:rlglue_interface}).
 \end{enumerate}
-
-We also discuss some sensible features of this environment, such as its stochasticity.
+The final part of this document discusses the different features of ALE, including the action stochasticity parameter and video recording.
 
 \section{Installing}\label{sec:install}
 
 \subsection{Requirements}
 
-The basic requirements to build and run the ALE are:
+The basic requirements to build and run ALE are:
 
 \begin{itemize}
-  \item \verb+CMake / make / g+++
+  \item \verb+CMake or make+
+  \item A C++ compiler
 \end{itemize}
-Notice that in the past (previous to version 0.5.0) the recommended method to compile the ALE was
-through Makefiles written by the ALE developers. This has changed and now we suggest everyone to
-use CMake to generate a Makefile and then to compile the code using such generated Makefile. The
-old Makefiles written for Linux and OS X (files \verb+makefile.mac+ and \verb+makefile.unix+) are
-still available, but we do not support them anymore.
-
-The ALE provides optional functionalities that are not activated by default, but if chosen the user is 
-required to use additional packages:
+
+Two custom makefiles are provided, \verb+makefile.mac+ (Mac OS X) and \verb+makefile.unix+ (Linux). However, beginning with version 0.5, we highly recommend the use of CMake rather than the custom makefiles.
```

Comment:
Here is the unique place you refer to the new version as 0.5. All the other places you refer to it as 0.5.0.